TechAndCheck/zenodotus
# Add youtube cop

Add custom YouTube style cop
Due to namespacing issues we have to stylize `YouTube` as `Youtube`.
This adds a custom Rubocop cop to enforce this.

To test:
In a random file add something like `YouTubeStuff.do_something` and then
run `rubocop` on the command line. It should flag the name. Remove the
line and run again and it should be gone.`
